### PR TITLE
Added data files

### LINKED ISSUE #15: Data file
Create a file which exports the needed data : 

- Each activity and how much it costs and how many liters per minute/time is used for this particular activity

### PR DESCRIPTION
**Pull request by SamiSha99**
Related to #11 #15
close #15 

Let's try this again.

### CODE REVIEW

**Comment by Karyum** [comment]
@SamiSha99 slow down i want you read the issue of the data file ok ? then check the pull request you just made is it correct or not correct depending on the issue?

File: src/App.jsx [hunk 1]
```diff
@@ -1,25 +1,11 @@
-import { BrowserRouter } from 'react-router-dom';
+import { Switch, BrowserRouter } from 'react-router-dom';
```
Comment (Amirk390) [inline src/App.jsx]: Removed any changes that has been made to a file that is not data please

**Review by Amirk390 — CHANGES_REQUESTED** [review]

**Review by Amirk390 — APPROVED** [review]
Looks good :100:

**Review by matthewdking — COMMENTED** [review]
@SamiSha99  In the design there is 6 activities and here you have 5 data items are you sure youre not missing an activity ?

**Comment by SamiSha99** [comment]
@matthewdking In the design there is 5 activities not 6, if you consider the flush size then it is considered an activity of 6, in that case, should I add the big flush as an activity?

**Comment by matthewdking** [comment]
@SamiSha99 I am looking at the prototype and can see six different types of activity

File: src/NavBar.jsx [hunk 2]
```diff
@@ -62,7 +62,11 @@ const animationBar = keyframes`
     margin: 0 0 0rem 0;
    }
   100% {
+<<<<<<< HEAD
+    height: 1.6rem;
+=======
     height: 2rem;
+>>>>>>> master
```
Comment (matthewdking) [inline src/NavBar.jsx]: This should not be here!

**Review by matthewdking — CHANGES_REQUESTED** [review]

**Review by Amirk390 — APPROVED** [review]
Looks good :+1: 
I see no conflict any more and the data looks ok

**Review by matthewdking — APPROVED** [review]
Nice :white_check_mark:

**Pull request merged**